recipes/iris/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set version = "1.9.2" %}
+
+package:
+    name: iris
+    version: {{ version }}
+
+source:
+    fn: iris-v{{ version }}.tar.gz
+    url: https://github.com/SciTools/iris/archive/v{{ version }}.tar.gz
+    sha256: 93f2cfe1c14aad293103727004863d09f21ea61de337aa73cb96a706df0b6133
+
+build:
+    number: 0
+    skip: True  # [osx]
```

Comment:
@pelson we have two issue on OS X.

- Python 2.7: downloads the most up-to-date `hdf5` and crashes due to a mismatch with `libnetcdf`. I am not sure if that is a bad pinning on our part or the same `conda-build` bug we experienced in RIOS. (Which I believe has a fix in the next relese.)

- Python 3: [`ImportError: No module named 'iris.fileformats._pyke_rules.compiled_krb'`](https://github.com/conda-forge/staged-recipes/pull/280#issuecomment-206377121)


I can confirm that adding the proper `hdf5` in the build requirements the first one, but I don't want to do that as it is a hack. The second one I have no idea. That is why I am skipping OS X here. If you are OK with that let's merge and fix that in the feedstock.